src/debugger/Breakpoint.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+export class Breakpoint {
+  //Will add more fields as needed in the future
```

Comment:
This is pretty much always true, so no need to spell it out.

Reply:
Removed